src/pages/console/auth/import.mdx

```diff
@@ -15,11 +15,11 @@ You can import existing Amazon Cognito resources into your Amplify project using
 
 To get started, Choose **Authentication** from the **Set up** menu in your app's Studio. On the **Authentication** page, choose **Reuse existing Amazon Cognito resources**.
 
-![import Amazon Cognito resource](/images/studio/import-auth.png)
+![Process to add authentication using a imported Amazon Cognito user pool resource](/images/studio/import-auth.png)
 
 Select a Cognito user pool and App clients required for your application.
 
-![Select user pool and app clients](/images/studio/import-auth-selection.png)
+![Selecting a user pool and app clients for a imported Amazon Cognito user pool resource](/images/studio/import-auth-selection.png)
```

Comment:
```suggestion
![Selecting a user pool and app clients for an imported Amazon Cognito user pool resource](/images/studio/import-auth-selection.png)
```

Reply:
The alt text for this is very similar to the text above which isn't that helpful when I'm using the screen reader. Could we either add more info to the paragraph above the image so that we can convey the steps shown in the image? That way we could make the image be decorative like this: `[](/images/studio/import-auth-selection.png)`

Do you have any thoughts about that @ykethan?

Reply:
make sense, let me make this decorative.